docker/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '2.1'
+services:
```

Comment:
what was the reason for removing `ltcd`?

Reply:
yeah, let us discuss this when we chat about `travis.yml`.